content/USER_GUIDE.md

```diff
@@ -0,0 +1,242 @@
+# **FLASHDeconv & FLASHTnT User Guide**
+
+Welcome to the **FLASHDeconv & FLASHTnT User Guide**. This guide provides a step-by-step walkthrough on using **FLASHDeconv** and **FLASHTnT**, from uploading data to processing and viewing results.
+
+---
+
+## **1️⃣ Uploading MS Data**
+
+1. Navigate to **FLASHDeconv > Workflow** in the sidebar.
+2. Click **"File Upload"** to upload your `.mzML` file.
+3. **Two options to add files:**
+   - **Drag and Drop** your file into the upload box.
+   - **Browse Files** to manually select the `.mzML` file.
+4. Click **"Add MS Data"** to confirm the upload.
+
+
+![Configure Parameters](/static/Images/flashdeconv_upload.png)
```

Comment:
The images still render as huge. You should be able to use inline html tags as follows:

`<img src="your-image.jpg" alt="Description" style="width:300px;">`